### PR TITLE
Fixed the conformance section showing up twice, added WebIDL text

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

**Comment by mounirlamouri** [comment]
This pull request do not apply anymore (I pushed something I wrote a week ago to fix the conformance text). Could you update the pull request with only the added text about WebIDL? (without the typos too ;))

**Comment by marcoscaceres** [comment]
@mounirlamouri fixed the merge conflict. However, I was not able to see what typos you were referring to?

**Comment by mounirlamouri** [comment]
Your PR contains 4 commits, is that on purpose?

**Comment by marcoscaceres** [comment]
@mounirlamouri yep. They are the ones listed above that address various things.

File: index.html [hunk 1]
```diff
@@ -85,10 +85,17 @@
...
-    <p>There is only one class of  product that can claim conformance to
-    this specification: a <dfn>user agent</dfn>.</p>
+     <p>There is only one class of  product that can claim conformance to 
+     this specification: a <dfn>user agent</dfn>.</p>
+     
+     <p>Implementations that use ECMAScript to implement the APIs defined
+        in this specification MUST implement them in a manner consistent 
+        with the ECMAScript Bindings defined in the Web IDL specification
+        [[!WEBIDL]], as this specification uses that specification and 
+        terminology.</p>
```
Comment (mounirlamouri) [inline index.html]: Could you re-use the DOM spec sentence?

File: index.html [hunk 2]
```diff
@@ -85,10 +85,17 @@
...
   </section>
 
+
```
Comment (mounirlamouri) [inline index.html]: and remove this new line?

**Comment by mounirlamouri** [comment]
You are breaking the indentation but I guess it's not big deal given that we are going to run tidy on the document.

**Pull request merged**